ruby-magic-link-example/views/index.erb

```diff
@@ -1,38 +1,52 @@
-  <div>
-      <img src="<%= url("/workos_logo_new.png")%>" alt="workos logo">
-  </div>
-</div>
-<div class='flex'>
-  <div class="logged_in_div_left">
-      <div>
-          <h1>Your app,</h1>
-          <h2>Enterprise Ready</h2>
-      </div>
-      <div>
-          <a href="https://workos.com/" target="_blank"><button class='button'>WorkOS</button></a>
-          <a href="https://workos.com/docs" target="_blank"><button class='button'>Documentation</button></a>
-          <a href="https://workos.com/docs/reference" target="_blank"><button class='button'>API Reference</button></a>
-          <a href="https://workos.com/blog" target="_blank"><button class='button'>Blog</button></a>
-          
-      </div>
-  </div>
+<div class="logged_in_nav">
+        <div class="flex">
+            <div>
+                <img src="<%= url("images/workos-logo-with-text.png")%>" alt="workos logo">
+            </div>
+            <div class="flex">
+                <p>You're logged in <%= @email %>, welcome!</p>
+            </div>
+        </div>
+        <div>
+            <a href="https://workos.com/docs" target="_blank"><button class='button nav-item'>Documentation</button></a>
+            <a href="https://workos.com/docs/reference" target="_blank"><button class='button nav-item'>API
+                    Reference</button></a>
+            <a href="https://workos.com/blog" target="_blank"><button
+                    class='button nav-item blog-nav-button'>Blog</button></a>
+            <a href="https://workos.com/" target="_blank"><button class='button button-outline'>WorkOS</button></a>
+        </div>
+    </div>
+    <div class='flex'>
+        <div class="logged_in_div_left">
+            <div class="title-text">
+                <h1>Your app,</h1>
+                <h2 class="home-hero-gradient">Enterprise Ready</h2>
+            </div>
+            <div class="title-subtext">
+                <p>Start selling to enterprise customers with just a few lines of code.</p>
+                <p>Implement features like single sign-on in minutes instead of months.</p>
+            </div>
+            <div class="flex success-buttons">
+                <a href="https://workos.com/signup" target="_blank"><button class='button'>Get Started</button></a>
+                <a href="mailto:sales@workos.com?subject=WorkOS Sales Inquiry" target="_blank"><button
+                        class='button button-outline sales-button'>Contact
+                        Sales</button></a>
+            </div>
+        </div>
   <div class="logged_in_div_right">
       <div class="flex_column">
           <h2>Raw Profile Response Details</h2>
           <div class="text_box">
-            <pre><%= @current_user %></pre>
+            <pre class="prettyprint"><%= @current_user %></pre>
           </div>
       </div>
       <a class='button logout_button' href='/logout'>Sign Out</a>
   </div>
 </div>
 
+
 <% else %>
   <div class="container_login">
     <div class='flex_column'>
```

Comment:
On the home page, pre-login, there are a couple styling issues. 
1. Could you use the transparent logo with the text here? 
2. The text input class isn't being applied for some reason
3. The button class padding needs to get cut in half 6/16 instead of 16/32
![CleanShot 2022-08-12 at 12 33 17](https://user-images.githubusercontent.com/47123183/184422685-334e84f5-1a7c-4172-8953-c879ba5f55b7.png)

Reply:
I can switch this over to the new transparent logo. 

In regards to the logo--could you check again using an incognito window? I think that this may be related to CSS caching that Sinatra does within the browser
<img width="1651" alt="CleanShot 2022-08-15 at 08 45 11@2x" src="https://user-images.githubusercontent.com/96083517/184668449-ae45dcbb-608b-4ffe-aeb6-bad6e77cf97e.png">